s3/variables.tf

```diff
@@ -42,5 +42,19 @@ variable "config" {
       body = string
       key  = string
     })))
+
+    # S3 to S3 data migration
+    data_migration = optional(object({
+      import = optional(object({
+        source_bucket_arn  = string
+        source_kms_key_arn = optional(string)
+        worker_role_arn    = string
+      }))
+      # todo: support the export side of data migration - replaces manual steps on source bucket side
+      # export = optional(object({
+      #   s3_migration_role_arn = string
+      # }))
```

Comment:
What is the plan for doing this? Is there another ticket?

Reply:
No plan... it felt wrong to support only the 'import' side of the equation when dataflow (account of the source bucket) will also be migrated onto DBT platform soon enough.  We haven't had a specific request to support it, but Dan Baark mentioned that maybe they would want it which is why we built in the import/export concept.  Do you think it is ok to create a ticket for something that hasn't been requested?

Reply:
@chopkinsmade?

Reply:
If it's not part of this ticket, I don't think we should leave this commented out code in.